ADD: Add blog post on SPC hail reports and NEXRAD

mgrover1 opened the pull request:
Closes #1157

zssherman commented:
@mgrover1 Only thing I see is some duplicate imports for pyart and cartopy, do we want to combine both import cells in the notebook or was there a reason why they were separate?

mgrover1 commented:
> @mgrover1 Only thing I see is some duplicate imports for pyart and cartopy, do we want to combine both import cells in the notebook or was there a reason why they were separate?

Yeah - I still need to clean this up more!

mgrover1 commented:
@russ-schumacher - I added you as a co-author here on the blog post! We extended your notebook a bit, added some more features to the maps, and consolidated some of the code.

mgrover1 commented:
Here is the animation if anyone is interested 

https://user-images.githubusercontent.com/26660300/203613091-49ec8fca-2f44-4af2-9f9e-02cdc4ad55d3.mov
